features/test/testService.js

```diff
@@ -0,0 +1,43 @@
+const dbConnection = require('../dbConnection');
+
+// Erstellen einer Service Klasse, die sich um CRUD Aktionen mit der Datenbank kümmert
+// C - Create
+// R - Read
+// U - Update
+// D - Delete
+// Achtung in JS gibt es keinen private deklarator bei Klassen
+class TestService {
+    constructor() {
+    }
+
+    get(succes) {
+        dbConnection.select("Select * FROM tests", succes);
```

Comment:
Would do all caps for SQL queries.
```suggestion
        dbConnection.select("SELECT * FROM tests", success);
```